irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -1377,6 +1411,33 @@ namespace iroha {
                              R"( WHERE (SELECT * FROM has_perm))",
                              R"(WHEN NOT (SELECT * FROM has_perm) THEN 2)"}});
 
+      statements.push_back(
+          {"removePeer",
+           removePeerBase,
+           {(boost::format(R"(
+            has_perm AS (%s),
+            get_peer AS (
+              SELECT * from peer WHERE public_key = $2
+              AND address = $3 LIMIT 1
+            ),
+            check_peers AS (
+              SELECT 1 WHERE (SELECT COUNT(*) FROM peer) > 1
```

Comment:
we could easily avoid this subquery
```suggestion
              SELECT COUNT(*) > 1 enough_peers FROM peer
```